src/components/ChatLog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+// const ChatLog = ({ entries, newEntry }) =>
+const ChatLog = (props) => {
+  // return entries.map((entry))
+  const chatEntries = props.entries.map((entry) => {
+    return (
+      <ChatEntry
+        key={entry.id}
+        id={entry.id}
+        sender={entry.sender}
+        body={entry.body}
+        timeStamp={entry.timeStamp}
+        liked={entry.liked}
+        newEntry={props.newEntry}
```

Comment:
newEntry feels a little misleading for a prop name because it doesn't quite convey that this is a function, but I was able to figure it out by reading line 26 in this file where i see `newEntry: PropTypes.func.isRequired` 

Perhaps newEntry could be named `onUpdateEntry`

You'd need to make this nane change in App.js and ChatEntry.js too